tst/gap/advanced/hermitianspreads.g

```diff
@@ -6,7 +6,7 @@ adj := function(x,y)
     return ProjectiveDimension(Meet(x,y))=-1;
 end;
 graph := Graph(coll,planes,OnProjSubspaces,adj,true);;
-cliques := CompleteSubgraphs(graph);;
+cliques := CompleteSubgraphs(graph,-1,2);;
```

Comment:
The tst file is (re)generated from this source, so I updated the code here now, too